src/shared/classes/NotificationsClasses.js

```diff
@@ -0,0 +1,62 @@
+import { cls } from '../cls.js';
+import { positionClass } from '../position-class.js';
+
+export const NotificationsClasses = (props, colors, classes) => {
+  const { opened } = props;
+  return {
+    base: {
+      common: cls(
+        positionClass('absolute', classes),
+        'z-50 transform transition-transform-opacity box-border',
+        !opened && '-translate-y-full opacity-0 pointer-events-none'
+      ),
+      ios: cls(
+        'rounded-xl px-2.5 py-2.5 start-2 end-2 top-2 shadow-lg',
+        colors.bgIos
+      ),
+      material: cls(
+        'rounded-2xl py-5 px-4 start-4 end-4 top-4',
+        colors.bgMaterial
+      ),
+    },
+    content: {
+      common: '',
+      ios: 'mt-2.5',
+      material: 'mt-2 ml-[calc(16px+16px)]',
```

Comment:
What is this a magic number? What if there is no icon? Icon should move the content when it is there